components/sidebar/index.jsx

```diff
@@ -1,9 +1,20 @@
 import React from 'react'
 import './index.css'
+import Logo from '../../design/logo'
 
 const Sidebar = React.memo(({ className }) => {
   return (
     <nav className={className}>
+      <div className="branding">
```

Comment:
It's actually a `<header>` and maybe even an AppBar but I am not sure yet 🙂

We have something similar in CORE Discovery browser-extension. Maybe you have better ideas.

![CORE Discovery Sidebar](https://user-images.githubusercontent.com/8440244/69803215-25a4e280-11e4-11ea-9d51-a33848a2f2e4.png)

Reply:
I am not sure if I understand it. Like this element should be in nav instead it should be in `header`/`appBar` component?

Reply:
No, not really. You have no restriction to count of `<header>`, `<nav>`, `<section>` and other semantic elements on the page.

Also, you have no restriction where to use `AppBar` component. You can do it everywhere it fits your needs.

On the other side, I feel like this one might be a fake DRY so I would not change it for this time. Sorry, I am confusing.